src/main/java/com/marklogic/kafka/connect/DefaultDatabaseClientConfigBuilder.java

```diff
@@ -1,14 +1,40 @@
 package com.marklogic.kafka.connect;
 
+/*
+ * v1.2.2 changes - START
+ */
```

Comment:
As noted below too, I recommend not including these comments. The set of imports will change over time, and the comments will soon not be accurate. And it's not worth spending time trying to maintain these comments - they just don't add any value.

Reply:
Will make the changes